docs/key_import.md

```diff
@@ -117,7 +117,8 @@ default until a key has been imported into them.
 * `external_key_count`: The number of external addresses generated.
 * `internal_key_count`: The number of change addresses generated.
 * `watch_only`: Whether the wallet has private key information for the account.
-  This is always true for `lnd`'s default wallet accounts.
+  `lnd`'s default wallet accounts always have private key information, so this
```

Comment:
If we're going to touch this: with the new `lncli createwatchonly` command it's possible to instantiate a node completely watch-only. In that case this boolean will turn `true`. Maybe link to the [remote signing](https://github.com/lightningnetwork/lnd/blob/master/docs/remote-signing.md) doc for more information?

Reply:
Sounds reasonable; since a second approver is needed will let them be arbiter